### DISCUSSION TITLE
CKAN - Home Page Frontend Developing

### DISCUSSION THREAD

**Discussion by idhamidregioit** [ckan/ckan#6942] [General]
Good Morning CKAN-Family,
I hope you will get this message in well.
I have installed CKAN on Ubuntu Server, Now I want to change the home page layout-Frontend (CSS/Styling). I have seen the documentation but there is very limited information. Can anyone one guide me from where I can start?

**Comment by disl-pdavid** [comment]
Hi!  I've done some rudimentary styling changes directly through the settings interface (/ckan-admin/config).

If it's just CSS you want to work with this is an easy way to make those changes.

If you wanted something more in-depth it looks like you'd want to see the docs on theming: https://docs.ckan.org/en/latest/theming/index.html.  This looks like it would require a plugin to override default templates to render your own (it's all pretty simple in Jinja).

(I'm literally about to start hacking at a theme myself so it was at the forefront of my mind... :) ).
